corehq/apps/fixtures/dbaccessors.py

```diff
@@ -27,6 +27,18 @@ def get_fixture_data_types_in_domain(domain):
     ))
 
 
+@quickcache(['domain', 'data_type_id'], timeout=60, skip_arg='bypass_cache')
+def get_fixture_items_by_data_type(domain, data_type_id, bypass_cache=False):
```

Comment:
Actually thinking about it do you know if the skip_arg busts the cache? if so then it should be fine because when you update items it passes the bypass_cache=True